src/common/logger/ShortlistingLogger.ts

```
@@ -450,4 +442,404 @@ export class ShortlistingLogger {
+      }
+
+      // Check if file exists to determine if we need to write headers
+      const fileExists = fs.existsSync(logFilePath);
+
+      // Prepare CSV line with proper comma delimiters
+      const csvLine = [
+        ShortlistingLogger.escapeCSV(enrollmentSuccessData.dateTime),
+        ShortlistingLogger.escapeCSV(enrollmentSuccessData.userId),
+        ShortlistingLogger.escapeCSV(enrollmentSuccessData.cohortId),
+        ShortlistingLogger.escapeCSV(enrollmentSuccessData.courseId),
+        ShortlistingLogger.escapeCSV(enrollmentSuccessData.enrollmentId || "")
```

ESLint

[error] Insert `,`

(prettier/prettier)